internal/enums/errors/errors.go

```diff
@@ -30,7 +30,8 @@ var ErrDockerNotInstalled = errors.New("{HORUSEC_CLI} Error Docker not found. Pl
 
 // Occurs when not found rails project
 
-var ErrNotFoundRailsProject = errors.New("{HORUSEC_CLI} Error not found rails project syntax")
+var ErrNotFoundRailsProject = errors.New("{HORUSEC_CLI} Brakeman only function on Ruby On Rails project." +
+	" Found no rails project syntax")
```

Comment:
I think that would be better "Brakeman only works on Ruby On Rails project."

Reply:
Also I think that `{HORUSEC_CLI}` is not necessary, since the log already have `HORUSEC_CLI` prefix, this would be redundant.